asapdiscovery-data/asapdiscovery/data/dask_utils.py

```diff
@@ -286,7 +286,12 @@ class LilacGPUDaskCluster(LilacDaskCluster):
     cores = 1
 
     @classmethod
-    def from_gpu(cls, gpu: GPU = GPU.GTX1080TI, silence_logs: int = logging.DEBUG):
+    def from_gpu(
+        cls,
+        gpu: GPU = GPU.GTX1080TI,
+        silence_logs: int = logging.INFO,
+        walltime: str = "72h",
```

Comment:
should this be passed into the constructor below?

Reply:
Indeed it should :eagle: :eyes: